systemvm/agent/noVNC/core/rfb.js

```diff
@@ -506,13 +521,46 @@ export default class RFB extends EventTargetMixin {
 
         const scancode = XtScancode[code];
 
+        if (keysym === KeyTable.XK_Shift_L || keysym === KeyTable.XK_Shift_R) {
+            this._shiftPressed = down;
+            this._shiftKey = down ? keysym : KeyTable.XK_Shift_L;
+        }
+        if (keysym === KeyTable.XK_Control_L || keysym === KeyTable.XK_Control_R) {
```

Comment:
Seems like ctrlPressed and altPressed are not used, can we remove these 2 checks?

Reply:
@nvazquez yes, I will remove them